docs/src/mpolynomial.md

```diff
@@ -217,13 +217,13 @@ x^2*y^2+x+1//1
 
 ```
 
-Incase a specific parent ring is constructed, it can also be passed to the function.
+In case a specific parent ring is constructed, it can also be passed to the function.
 
 ```
 
 **Examples**
 
-```jldoctest
+```@jldoctest
```

Comment:
Why the `@` here? I don't remember having seen this form, and I didn't find it documented.

Reply:
Hmm, I assumed that's why it wasn't rendering. But apparently I am wrong. Do you see what is wrong here?

Reply:
It's ok, I found it. I just pushed a proper fix.